scripts/build.sh

```diff
@@ -15,6 +15,11 @@ python3 -m venv ${REPO_BASE_DIR}/target/venv
 # Install dependencies
 ${REPO_BASE_DIR}/target/venv/bin/pip install -r ${SCRIPTS_DIR}/requirements/requirements.txt
 
+# Check if the dependencies were installed successfully
+if ! command -v jsonschema > /dev/null 2>&1; then
+  echo "jsonschema is required. Please install!"
+  exit 1
+fi
```

Comment:
This is not required. We install `jsonschema` automatically here: https://github.com/mesosphere/universe/blob/version-3.x/scripts/requirements/requirements.txt#L1 which is executed in the line 16 above. I'll create a PR.